server/middlewares/authorizeUser.js

```diff
@@ -0,0 +1,20 @@
+import models from '../database/models';
+import responseMessage from '../helpers';
+
+const { UserRole, Role } = models;
+
+export default async (request, response, next, roleArray) => {
+  try {
+    const { id } = request.userData;
+    const userrole = await UserRole.findOne({
+      where: { userId: id },
+      include: [Role]
+    });
+    if (!userrole) responseMessage(response, 403, { message: 'UnAuthorized User. Role not found' });
```

Comment:
I think the convention is to make the message lower case

Reply:
done